### ISSUE TITLE
Incorrect work of drag-and-drop with ng-container in ivy

### ISSUE THREAD

**Issue by nmatushevskiy** [angular/components#17505]
#### Reproduction

repo for reproducing: https://github.com/nmatushevskiy/ivy-ng-container-bug-repro/

Steps to reproduce:
1. npm i
2. ng serve
3. Try dragging any item
 
 
#### Expected Behavior
Items being dragged correctly with using ng-container
https://stackblitz.com/github/nmatushevskiy/ivy-ng-container-bug-repro
(example without ivy)

Code example:
```
<div cdkDropList cdkDropListOrientation="horizontal" class="example-list" (cdkDropListDropped)="drop($event)">
  <ng-container *ngFor="let item of items">
    <div class="example-box" cdkDrag>{{item}}</div>
  </ng-container>
</div>
```

#### Actual Behavior
Dragging works incorrectly with using ng-container and ivy.

With this code:
```
<div cdkDropList cdkDropListOrientation="horizontal" class="example-list" (cdkDropListDropped)="drop($event)">
  <ng-container *ngFor="let item of items">
    <div class="example-box" cdkDrag>{{item}}</div>
  </ng-container>
</div>
```

Have the following behavior:
![Снимок экрана 2019-10-25 в 17 49 51](https://user-images.githubusercontent.com/56971144/67580994-e8b27f80-f74f-11e9-9c1e-08692577b30b.png)

And this, if I try to drag a few items:
![Снимок экрана 2019-10-25 в 17 49 40](https://user-images.githubusercontent.com/56971144/67581009-ed773380-f74f-11e9-9b8e-f941369a2008.png)

#### Environment

- Angular: 9.0.0-next.13
- CDK/Material: 8.2.3 / 9.0.0-next.0 (I tried both)
- Browser(s): Chrome v. 78.0.3904.70
- Operating System: macOS

#### Workaround
drag and drop works correct without ng-container.
For example:
```
<div cdkDropList cdkDropListOrientation="horizontal" class="example-list" (cdkDropListDropped)="drop($event)">
    <div *ngFor="let item of items" class="example-box" cdkDrag>{{item}}</div>
</div>
```

#### Comment
By debugging established:
in `drop-list.ts` (https://github.com/angular/components/blob/master/src/cdk/drag-drop/directives/drop-list.ts)
in `ngAfterContentInit` method there are no items:
![Снимок экрана 2019-10-25 в 18 32 07](https://user-images.githubusercontent.com/56971144/67584542-92950a80-f756-11e9-9f41-0207eeffb72a.png)
While without ivy all items received:
![Снимок экрана 2019-10-25 в 18 35 36](https://user-images.githubusercontent.com/56971144/67584683-d6880f80-f756-11e9-9f0c-79e61b04f344.png)

Also, with ivy doesn't work output `cdkDropListDropped`. By comparing code with and without ivy it was found out:
in `drag-ref.ts` (https://github.com/angular/components/blob/master/src/cdk/drag-drop/drag-ref.ts)
in method `_endDragSequence`, `this._dragDropRegistry.isDragging` returns false, so the code below is not being executed. Thats why the output is not called.
![Снимок экрана 2019-10-25 в 18 48 33](https://user-images.githubusercontent.com/56971144/67585959-3bdd0000-f759-11e9-9a6e-187bb6898889.png)

While without ivy `this._dragDropRegistry.isDragging` returns true and output called correctly
![Снимок экрана 2019-10-25 в 18 58 54](https://user-images.githubusercontent.com/56971144/67586131-8e1e2100-f759-11e9-9d6f-79f301c2e47c.png)

**Comment by crisbeto** [comment]
Fixed in https://github.com/angular/components/pull/17226 which hasn't been released yet.

**Comment by crisbeto** [comment]
Looks like the fix made it into 9.0.0-next.1.
